newest/components.cpp

```diff
@@ -65,6 +66,7 @@ CircleRender::CircleRender( GameObject* parent, double radius ) :
 radius ( radius ), Component(parent, string("CircleRender")) {}
 
 void CircleRender::update( float dt ) {
+  //printf("floatingUpdatedt:%f \n", dt);
```

Comment:
fixedUpdate seems to be working, but un-commenting this line, or adding a print dt statement to the component update method shows mostly 0.0000s or even negative values. Can you please resolve this before merging?